### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by nottherealalanturing**
## Describe your changes
Features added include

- Completed tasks could be marked.
- Completed tasks could be removed from the list and unfinished tasks will remain.
- Clear all tasks

### What kind of change does this PR introduce?

New Feature - Mark Completed Task, Clear Completed Tasks, Remove all tasks.

### In which branch do you want the changes to merge?

Main.

### Documentation updated ?

No

### Checklist

- [ ] No ES Lint Error
- [ ] No Style Lint Error
- [ ] Unit test case update

### Screenshots
Screen Shot 2022-04-28 at 18 20 06

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 
 const addInput = document.querySelector('.newItem');
+const clearCompletedBtn = document.querySelector('.clearBtn');
+const clearAll = document.querySelector('.fa-arrows-rotate');
 
 const editAction = () => {
   document.querySelectorAll('.task').forEach((val) => {
-    val.children[0].children[1].addEventListener('blur', (e) => {
+    val.children[1].children[0].addEventListener('blur', (e) => {
       /* eslint-disable */
```
Comment (micheaol) [inline src/index.js:14]: Please ensure that `linters` are not disabled. A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).

File: src/index.js [hunk 2]
```diff
@@ -58,14 +61,25 @@ const populateDOM = () => {
   const tasksList = document.querySelector('#tasks');
   let newList = '';
 
-  MyTasks.tasks.forEach((val) => {
```
Comment (micheaol) [inline src/index.js:61]: Your app is not performing as expected. I am unable to add a new task. Please find below the error displayed. Kindly fix it. As a result of the error, I am unable to checkout the status functionality. Kindly fix the error.

image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @nottherealalanturing , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (nottherealalanturing) [reply src/index.js:14]: My prettier breaks down long lines of code into multiple lines and that is what's been flagged by eslint hence the ignore.

File: src/index.js (hunk 2 above)
Comment (nottherealalanturing) [reply src/index.js:61]: Will address that

**Review by descholar-ceo — APPROVED** [review]
Hi @nottherealalanturing  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**